components/text/link.js

```diff
@@ -50,6 +58,18 @@ export const InternalLink = (
         color: #fff;
       }
     `}</style>
+    <style jsx>{`
+      span {
+        ${display ? `display: ${display};` : ''}
+        ${display === 'block' ? 'margin: 1em 0;' : ''}
```

Comment:
```suggestion
        margin: ${display === 'block' ? '1em 0' : 'inherit'};
```
This can cause invalid CSS in `styled-jsx`